.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Feature Request
+about: Create a proposal to request a feature
+labels: enhancement, needs-triage
+---
+
+<!-- < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < ☺ 
+v                            ✰  Thanks for opening an issue! ✰    
+v    Before smashing the submit button please review the template.
+v    Word of caution: poorly thought-out proposals may be rejected 
+v                     without deliberation 
+☺ > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > >  -->
+
+## Summary
+
+<!-- Short, concise description of the proposed feature -->
+
+## Problem Definition
+
+<!-- Why do we need this feature? 
+What problems may be addressed by introducing this feature?
+What benefits does ICS stand to gain by including this feature?
+Are there any disadvantages of including this feature? -->
+
+## Proposal
+
+<!-- Detailed description of requirements of implementation -->
+
+____
+
+#### For Admin Use
+
+- [ ] Not duplicate issue
+- [ ] Appropriate labels applied
+- [ ] Appropriate contributors tagged
+- [ ] Contributor assigned/self-assigned
+- [ ] Is a spike necessary to map out how the issue should be approached?
```

Comment:
Sorry, what is a "spike" in this context?

Reply:
IIUC,  https://en.wikipedia.org/wiki/Spike_(software_development)